exercises/D-logical-operators/exercise.js

```diff
@@ -9,16 +9,17 @@
 let htmlLevel = 8;
 let cssLevel = 4;
 
+
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = htmlLevel > 5;
+let cssLevelAbove5 = cssLevel > 5;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-let cssAndHtmlAbove5;
-let cssOrHtmlAbove5;
+let cssAndHtmlAbove5 = htmlLevelAbove5 && cssLevelAbove5;
+let cssOrHtmlAbove5 =  htmlLevelAbove5 || cssLevelAbove5;
 
```

Comment:
Hi Mo,
I would have suggest a code review on this exercise as you need to use its previous varible`htmlLevelAbove5` & `cssLevelAbove5` for comparison. 😊

Reply:
Hi Mo
Thank you, it was perfect